cfme/configure/configuration/server_settings.py

```diff
@@ -60,35 +62,31 @@ class basic_information(View):  # noqa
+        automate = BootstrapSwitch(name="server_roles_automate")
         ems_metrics_coordinator = BootstrapSwitch(name="server_roles_ems_metrics_coordinator")
-        ems_operations = BootstrapSwitch(name="server_roles_ems_operations")
         ems_metrics_collector = BootstrapSwitch(name="server_roles_ems_metrics_collector")
-        reporting = BootstrapSwitch(name="server_roles_reporting")
         ems_metrics_processor = BootstrapSwitch(name="server_roles_ems_metrics_processor")
+        cockpit_ws = BootstrapSwitch(name="server_roles_cockpit_ws")
+        database_operations = BootstrapSwitch(name="server_roles_database_operations")
+        embedded_ansible = BootstrapSwitch(name='server_roles_embedded_ansible')
+        event = BootstrapSwitch(name="server_roles_event")
+        git_owner = BootstrapSwitch(name="server_roles_git_owner")
+        notifier = BootstrapSwitch(name="server_roles_notifier")
+        ems_inventory = BootstrapSwitch(name="server_roles_ems_inventory")
+        ems_operations = BootstrapSwitch(name="server_roles_ems_operations")
+        reporting = BootstrapSwitch(name="server_roles_reporting")
         scheduler = BootstrapSwitch(name="server_roles_scheduler")
         smartproxy = BootstrapSwitch(name="server_roles_smartproxy")
-        database_operations = BootstrapSwitch(name="server_roles_database_operations")
         smartstate = BootstrapSwitch(name="server_roles_smartstate")
-        event = BootstrapSwitch(name="server_roles_event")
         user_interface = BootstrapSwitch(name="server_roles_user_interface")
         web_services = BootstrapSwitch(name="server_roles_web_services")
-        ems_inventory = BootstrapSwitch(name="server_roles_ems_inventory")
-        notifier = BootstrapSwitch(name="server_roles_notifier")
-        automate = BootstrapSwitch(name="server_roles_automate")
-        rhn_mirror = BootstrapSwitch(name="server_roles_rhn_mirror")
-        database_synchronization_role = BootstrapSwitch(
-            name="server_roles_database_synchronization")
-        git_owner = BootstrapSwitch(name="server_roles_git_owner")
-        websocket = BootstrapSwitch(name="server_roles_websocket")
-        cockpit_ws = BootstrapSwitch(name="server_roles_cockpit_ws")
-        # STORAGE OPTIONS
-        storage_metrics_processor = BootstrapSwitch(name="server_roles_storage_metrics_processor")
-        storage_metrics_collector = BootstrapSwitch(name="server_roles_storage_metrics_collector")
-        storage_metrics_coordinator = BootstrapSwitch(
-            name="server_roles_storage_metrics_coordinator")
-        storage_inventory = BootstrapSwitch(name="server_roles_storage_inventory")
-        vmdb_storage_bridge = BootstrapSwitch(name="server_roles_vmdb_storage_bridge")
+        # The websocket role no longer exists in 5.11.
+        # Instead of creating a whole new View, set it to None so that
+        # read() and fill() skip it.
```

Comment:
Yup, if you set `None` it will work, but we not implement view in such way... I think better to handle this in `respective method` or `test`.

Reply:
"I think better to handle this in respective method or test." Can you clarify what you mean by this?

Reply:
you already removed `None` usecase :+1: that great... I was talking to handle this situation in method where this view will use.